hr_timesheet_sheet/models/hr_timesheet_sheet.py

```diff
@@ -320,9 +338,13 @@ def create(self, vals):
                 raise UserError(
                     _('In order to create a sheet for this employee, '
                       'you must link him/her to an user.'))
+        line_ids = self.env['hr_timesheet.sheet.line']
+        if 'line_ids' in vals:
+            for val in vals['line_ids']:
```

Comment:
@mreficent Here ` for val in vals.get('line_ids', False):`
No need to check `if 'line_ids' in vals:`